frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -50,28 +59,23 @@ class OverrideForkPointDialog(
         )
       )
     }
+
```

Comment:
Re-posting so that it's not lost:

> I'd suggest another thing however (can be done within a separate issue): to display, within the Override Fork Point dialog, the list of commits that will be considered unique to the branch, assuming the selected fork point... WDYT?

Reply:
IIUC that would be a new look - not something we want to simply add to the present dialog, right?

anyway, instead of re-posting new ideas (or more critical changes) in resolvable and collapsable comments within a PR it is better to discuss them within the related issue or to open a new one, so that it's not lost 🙂

Reply:
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1279